### DISCUSSION TITLE
Multi Tenant Support?

### DISCUSSION THREAD

**Discussion by nimeshvaghasiya** [clerk/javascript#735] [Support request - DO NOT OPEN HERE] [answered]
How clerk-auth supports multi-tenancy SignUp and SignIn?
I really need to create a tenant signup instead a normal user  signup?

**Comment by SokratisVidros** [comment, marked as answer]
Hi @nimeshvaghasiya. 

Clerk supports Gmail-like experience via [MultiSessionApplications](https://clerk.dev/docs/authentication/multi-session-applications). That is, your application can allow multiple users to sign-in from the same client.

Let me know if this helps.
